style/ruby/README.md

```diff
@@ -1142,6 +1152,188 @@ def some_method_with_parameters(param1, param2)
 end
 ```
 
+# Classes e Modulos
+## Consistência nas classes
+
+Use uma estrutura consistente em suas definições de classe.
+
+```ruby
+class Person
+  # Estenda (extend) e inclua (include) vêm primeiro
+  extend SomeModule
+  include AnotherModule
+
+  # Classes internas (StandardError)
+  CustomError = Class.new(StandardError)
+
+  # Constantes são as próximas
+  SOME_CONSTANT = 20
+
+  # Em seguida, temos atributos (se houver)
+  attr_reader :name
+
+  # Seguidos por validaçõe (se houver)
+  validates :name
+
+  # Métodos de classe públicos vêm em seguida
+  def self.some_method
+  end
+
+  # Inicialização (initialize) vem entre métodos de classe e outros métodos de instância
+  def initialize
+  end
+
+  # Seguido por outros métodos de instância públicos
+  def some_method
+  end
+
+  # Métodos protegidos (protected) e privados (private) são agrupados no final.
+  protected
```

Comment:
Nesse agrupamento seria bom pular duas linhas né, fica mais tcham associar o início e fim do agrupamento:
Ex:

```ruby
class GominhoDelicia
  def initialize
    #
  end


  protected

  def foo
    #
  end

  def foo2
  end


  private

  def foo3
    #
  end
end
```

Reply:
@felipegomesdev não ha necessidade de pular duas linhas. como time conversamos sobre isso. mas podemos perguntar a todos os outros devs. o que vcs acham?
@gildemberg-santos @lucaspontarolo @felnente @AndreFSilveira @VagnerOliveira1 @felnente @euclidesdry

Reply:
Acredito que uma única linha pode resolver, pois se a classe estiver sobrecarregada com muitos métodos, os dois espaços não ajudarão tanto. Nesse caso, seria recomendado desacoplar as funcionalidades dentro da classe.